site/en/guide/tpu.ipynb

```diff
@@ -166,7 +168,7 @@
         "id": "DcDPMZs-9uLJ"
       },
       "source": [
-        "First, creates the `TPUStrategy` object."
+        "To demonstrate this, create a `TPUStrategy` object:"
```

Comment:
Use full symbol name for `TPUStrategy`

Reply:
👍

`tf.distribute.TPUStrategy` <-`TPUStrategy`